detectors/azure/azurevm/README.md

```diff
@@ -0,0 +1,3 @@
+# Azure VM Resource detector
+
+[![PkgGoDev](https://pkg.go.dev/badge/go.opentelemetry.io/contrib/detectors/azure/azurevm)](https://pkg.go.dev/go.opentelemetry.io/contrib/detectors/azure/azurevm)
```

Comment:
This URL doesn't yet exist. Should the README only be added after the package is published?